web/templates/home.html.tmpl

```diff
@@ -4,4 +4,113 @@
     <h1>{{ .Title }}</h1>
     <p>{{ .Paragraph }}</p>
 </div>
+<div class='container'>
```

Comment:
The `container` class will add some max-width and extra margins and it only should be used once.

We usually divide the content of a section in `<section>  </section>` that has no extra style and it won't push the content related to the other elements (titles or descriptions)